index.html

```diff
@@ -126,56 +117,97 @@ <h2 class="tonic">Multi-Post Stories</h2>
+            <li class="i"><a href="#"><img src="images/twitter.png" alt=""></a></li>
+            <li class="i"><a href="#"><img src="images/github.png" alt=""></a></li>
+            <li class="i"><a href="#"><img src="images/medium.png" alt=""></a></li>
+          </ul>
+          <div class="Job-btn">
+            <button type="button" name="button" class="Job2-button"> Get my resume</button>
+          </div>
+        </section>
+
+        <section class="Skills">
+          <ul class="skill">
+            <li class="lang">
+              <h2>Languages</h2>
+              <img src="images/Union.png" alt="">
+            </li>
+            <li class="lang2">
+              <ul class="skills">
+                <li class="sk"><img src="images/Ellipse.png" alt=""><h4>JavaSript</h4></li>
+                <li class="sk"><img src="images/Ellipse1(1).png" alt=""><h4>HTML</h4></li>
+                <li class="sk"><img src="images/Ellipse1(2).png" alt=""><h4>CSS</h4></li>
+              </ul>
+            </li>
+          <li class="lang1">
+            <h2>Frameworks</h2>
+            <img src="images/Union1.png" alt="">
           </li>
-          <li class="lang2">
-            <ul class="skills">
-              <li class="sk"><img src="images/Ellipse.png" alt=""><h4>JavaSript</h4></li>
-              <li class="sk"><img src="images/Ellipse1(1).png" alt=""><h4>HTML</h4></li>
-              <li class="sk"><img src="images/Ellipse1(2).png" alt=""><h4>CSS</h4></li>
-            </ul>
+          <li> <hr> </li>
+          <li class="lang1">
+            <h2>Skills</h2>
+            <img src="images/Union1.png" alt="">
           </li>
-        <li class="lang1">
-          <h2>Frameworks</h2>
-          <img src="images/Union1.png" alt="">
-        </li>
-        <li class="lang1">
-          <h2>Skills</h2>
-          <img src="images/Union1.png" alt="">
-        </li>
-      </ul>
-      </section>
+          <li> <hr> </li>
+        </ul>
+        </section>
+
+      </div>
+
+
+<!-- THE FORM -->
       <section class="form">
         <h2>Contact me</h2>
         <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
         <form name="form1" action="https://formspree.io/f/myyovzdk" method="post">
           <label >
-            <input type="text" name="name" id="name"  placeholder="Yeremias" maxlength="30" required>
+            <input type="text" name="name" id="name"  placeholder="Your name" maxlength="30" required>
           </label> <br>
           <label >
            <input type="email" name="email-address" id="email" placeholder="natajayanj14@gmail.com" maxlength="30" required>
           </label>
           <textarea name="comment" rows="8" cols="80" placeholder="Write your message here" maxlength="500" required></textarea>
+          <br>
           <button type="submit" name="submit" class="btn3">  Get It Touch </button>
```

Comment:
- [ ] OPTIONAL: It's a nice to apply these buttons effects: `:hove`, `:active`, `:disabled`, etc. This will help users in case of accessibility and design reasons.
- [ ] OPTIONAL: This is a plus for the best form design:
```
form inputs {
   border: none;
  outline: none;
}
````
Try it and you'll see 😄. It will remove all black borders from your inputs.